### ISSUE TITLE
Option legacy image selection does not modifies the image.

### ISSUE THREAD

**Issue by M-um-p-i-t-z** [Tommy-Geenexus/exif-eraser#64]
phone: oneplus10Pro
Android: 13 

If I use this option pick a picture it allways say 0 to 1 
It is not modified or saved.
I would like to use the auto delete option, too, but it fails in first step.
If I could provided any morw Information, please tell me what you need. Thx.

**Comment by Tommy-Geenexus** [comment]
Have you verified that the images you intend to modify contain metadata?
Does it fail with legacy selection only?

**Comment by M-um-p-i-t-z** [comment]
> Have you verified that the images you intend to modify contain metadata?
> Does it fail with legacy selection only?

Hello,
yes it was only with legacy selection and yes fresh shooting pictures, but the issue has gone, also the auto delete now works. 
Seems that I messed up it, but I did not see white mices along the floor running, I swear.  It tried it several times after shooting pictures and it does fails by first step (removing tags) with error message. Next time if it happen again so, I will make screenhots.

I also do not exclude that I had set the permissions incorrectly.

I think can be closed for now, just tried it several times everything goes now!

Thanks a lot!
bye
